interface/weno/drug-drug.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
+$fake_register_globals=false;
+$sanitize_all_escapes=true;
+require_once("../globals.php");
+
+/*
+*   check to see if RxNorm installed
+*/ 
+	$rxn = sqlQuery("SELECT table_name FROM information_schema.tables WHERE table_name = '" . mitigateSqlTableUpperCase('RXNCONSO');
+if($rxn == false){
+	
+	die(xlt("Could not find RxNorm Table! Please install."));
+} 
+/*
+*   Grab medication list from prescriptions list
+*   load into array
+*/
+ $medList = sqlStatement("SELECT drug FROM prescriptions WHERE active = 1 AND patient_id = ?", array($pid));
+ $nameList = array();
+while($name = sqlFetchArray($medList)){
+	$drug = explode(" ", $name['drug']);
+	$rXn = sqlQuery("SELECT `rxcui` FROM `" . mitigateSqlTableUpperCase('RXNCONSO') . "` WHERE `str` LIKE ?", array("%" . $drug[0] . "%"));
+	$nameList[] = $rXn['rxcui'];
+}
+/*
+*  make sure there are drugs to compare
+*/
+$n = count($nameList);
+
+if (count($nameList) < 2) {
+    echo xlt("Need more than one drug.");
+    exit;
```

Comment:
Weird how some systems will throw errors and others won't.  But I added the closing bracket.